gradle-plugin/src/main/kotlin/arrow/meta/plugin/gradle/ArrowKotlinGradleSubplugin.kt

```diff
@@ -28,11 +28,19 @@ class ArrowKotlinGradleSubplugin : KotlinGradleSubplugin<AbstractCompile> {
     variantData: Any?,
     androidProjectHandler: Any?,
     kotlinCompilation: KotlinCompilation<KotlinCommonOptions>?
-  ): List<SubpluginOption> = emptyList()
+  ): List<SubpluginOption> {
+
+    @Suppress("UNUSED_VARIABLE")
+    val extension = project.extensions.findByType(ArrowExtension::class.java)
```

Comment:
I can't really tell from the code for what this is used. Can you help me, please ? @jereksel

Reply:
When we will be adding options for gradle plugin we must pass them to compiler plugin. It's done in this method

Reply:
But there are no Options here now. So why add a variable?

Reply:
So it'll be easier to add options in the future. I don't want to remove something that will surely be used in the future.

Reply:
We need settings soon for all plugins and meta itself to enable and disable plugins via config without code changes so if this provides an entry path to it let's keep it.